index.ios.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { View, TextInput, Button, StyleSheet, AppRegistry } from 'react-native';
+import Header from './src/components/header';
+import Inputs from './src/components/inputs';
+import SubmitArea from './src/components/submitArea';
+
+class App extends Component {
```

Comment:
index.ios.js and index.android.js should better use one App component instead of code copypasting